tasks/db_deploy/README.md

```diff
@@ -67,8 +67,8 @@ standards expected.
    ```bash
    touch migrations/migrate_versions_4_to_5/migrate.py
    touch migrations/migrate_versions_4_to_5/migrate_sql.py
-   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_v5.py
-   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_sql_v5.py
+   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_v6.py
+   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_sql_v6.py
```

Comment:
I think this should not be removed. 
The new files should be added: 
test/unit_tests/migrations/migrate_versions_5_to_6/test_migrate_v6.py
   touch test/unit_tests/migrations/migrate_versions_5_to_6/test_migrate_sql_v6.py

Reply:
Changed to parameterized names to better fit the example.